apps/api/src/graphql/schema/programs.ts

```diff
@@ -66,10 +66,14 @@ input ProgramRequirementsQuery {
     programId: String!
 }
 
+input ProgramQuery {
+    id: String!
```

Comment:
I argued earlier that this should actually be three different types, and I stand by that now. By combining them into one type you lost even more meaning about the semantics of the `id` field, which really does behave differently among these three resources.